R/pkg/R/mllib_clustering.R

```diff
@@ -610,3 +616,58 @@ setMethod("write.ml", signature(object = "LDAModel", path = "character"),
           function(object, path, overwrite = FALSE) {
             write_internal(object, path, overwrite)
           })
+
+#' PowerIterationClustering
+#'
+#' A scalable graph clustering algorithm. Users can call \code{spark.assignClusters} to
+#' return a cluster assignment for each input vertex.
+#'
+#  Run the PIC algorithm and returns a cluster assignment for each input vertex.
+#' @param data A SparkDataFrame.
+#' @param k The number of clusters to create.
+#' @param initMode Param for the initialization algorithm.
+#' @param maxIter Param for maximum number of iterations.
+#' @param sourceCol Param for the name of the input column for source vertex IDs.
+#' @param destinationCol Name of the input column for destination vertex IDs.
```

Comment:
nit. Here, `Name` -> `Param for the name` for consistency with the other param descriptions?

Or, is it better to remote `Param for` prefix in other descriptions?